api/src/main/java/org/openmrs/module/patientqueueing/model/PatientQueue.java

```diff
@@ -0,0 +1,150 @@
+package org.openmrs.module.patientqueueing.model;
+
+
+import org.openmrs.Patient;
+import org.openmrs.Provider;
+import org.openmrs.Location;
+import org.openmrs.Encounter;
+import org.openmrs.BaseOpenmrsData;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+import javax.persistence.Id;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Column;
+import javax.persistence.ManyToOne;
+import javax.persistence.JoinColumn;
+import java.io.Serializable;
+
+@Entity(name = "patientqueueing.PatientQueue")
+@Table(name = "patient_queue")
+public class PatientQueue extends BaseOpenmrsData implements Serializable {
+
+    @Id
+    @GeneratedValue
+    @Column(name = "patient_queue_id")
+    private Integer patientQueueId;
+
+    @ManyToOne
+    @JoinColumn(name = "patient_id")
+    private Patient patient;
+
+    @ManyToOne
+    @JoinColumn(name = "provider_id")
```

Comment:
Is the the provider that the patient is queued to see?

Reply:
Yes